house/views.py

```diff
@@ -31,7 +33,26 @@ def global_page(request):
 
 
 def house_login(request):
-    raise NotImplementedError
+    errorMsg = ""
```

Comment:
I think that this line is unnecessary

Reply:
> I think that this line is unnecessary

It is necessary, because if the house_id that the user have inserted doesn't exist in the data base, this string is used, and prints the inserted id (so the user can check if he has inserted it correctly)